### PR TITLE
Remove aws lambda dependency

### PR DESCRIPTION
**Pull request by sisisin**
Now, creating badge complete just cli and s3 bucket.
Zoi has been changed to a simple command line tool.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,50 +1,100 @@
-'use strict'
```
Comment (akiakaba) [inline src/index.js:1]: Did this file need chmod?

File: src/read-badge-json.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+const fs = require('fs');
+const path = require('path');
+
+module.exports = (jsonPath) => {
+  return new Promise((resolve, reject) => {
+    fs.readFile(path.resolve(process.cwd(), jsonPath), (err, data) => {
+      if(err) return reject(err);
```
Comment (akiakaba) [inline src/read-badge-json.js]: [nits] white-space ;)

File: src/index.js [hunk 3]
```diff
@@ -1,50 +1,100 @@
...
+  }
 };
+
+function putToS3(badgeData, badgeParam) {
+  const aws = require('aws-sdk');
+  const s3 = new aws.S3({ apiVersion: '2006-03-01' });
+  const imageType = badgeParam.imageType;
+  const contentType = contentTypeOf(imageType);
+
+  const putParam = getConfig(getArgs().config);
```
Comment (akiakaba) [inline src/index.js]: It seems to me that `putToS3` takes [`args`](https://github.com/opt-tech/zoi/pull/7/files#diff-1fdf421c05c1140f6d71444ea2b27638R9) as a parameter.

**Review by akiakaba — COMMENTED** [review]
cool

File: example/zoi.config.js [hunk 4]
```diff
@@ -0,0 +1,6 @@
+module.exports = () => {
+  return {
+    bucket: 'zoi-public',
+    path: 'branch/'
```
Comment (ocadaruma) [inline example/zoi.config.js]: I think that term `key` is more appropriate than `path`

Comment (ocadaruma) [inline example/zoi.config.js]: Why badge config and s3 config are separated?

File: README.md [hunk 5]
```diff
@@ -1,10 +1,8 @@
...
 
-# self deploy
-
-```
-$ export AWS_ACCESS_KEY_ID=[access_key]
-$ export AWS_SECRET_ACCESS_KEY=[secret_key]
-$ bash ./deploy.sh
+```bash
+$ npm install -g @sisisin/zoi-cli
+$ zoi --jsonFile ./example/zoi-cov.json --config ./example/zoi.config.js
```
Comment (ocadaruma) [inline README.md:9]: It seems aws credentials are still required to put to S3.

File: src/index.js (hunk 1 above)
Comment (sisisin) [reply src/index.js:1]: It is not necessary.

File: src/index.js (hunk 3 above)
Comment (sisisin) [reply src/index.js]: `getArgs()` returns the same value each time.
And I want to keep fewer arguments of functions.
So, I prefer to get when args become necessary.

**Comment by akiakaba** [comment]
Thank you!

**Pull request merged**